config/local.neon.dist

```diff
@@ -0,0 +1,11 @@
+# Host Config
+parameters:
+
+	# Database
+	database:
+		driver: pdo_pgsql
```

Comment:
Nemuzes zmenit obsah local.neon.dist. Tam musi byt to co tam bylo.

Reply:
Hlavne pdo_sqlite, to dela ten problem.

Reply:
už jsem to sám zjistzil